### PR TITLE
app-laptop/thinkfan: bump to version 1.0.1

### PR DESCRIPTION
**Pull request by ConiKost**
Package-Manager: Portage-2.3.52, Repoman-2.3.12
Signed-off-by: Conrad Kostecki <conrad@kostecki.com>

@Zlogene: Since you added yourself into metadata.xml, is it okay, to co maintain by myself? Since I also said on the ML, I would like to maintain.

### CODE REVIEW

File: app-laptop/thinkfan/metadata.xml [hunk 1]
```diff
@@ -1,11 +1,27 @@
...
+	</maintainer>
+	<longdescription>
+		Thinkfan is a simple, lightweight fan control program.
+		Originally designed specifically for IBM/Lenovo Thinkpads,
+		it now supports any kind of system via the sysfs hwmon interface.
+		It is designed to eat as little CPU power as possible.
+	</longdescription>
 	<use>
-		<flag name="atasmart">include libatasmart support to get disc temperature</flag>
+		<flag name="atasmart">Use <pkg>dev-libs/libatasmart</pkg> to read temperatures directly from hard disks.</flag>
```
Comment (juippis) [inline app-laptop/thinkfan/metadata.xml]: These descriptions don't usually have dots in the end.

File: app-laptop/thinkfan/metadata.xml [hunk 2]
```diff
@@ -1,11 +1,27 @@
...
+		<flag name="nvml">Allows to read GPU temperatures from the proprietary nVidia driver <pkg>x11-drivers/nvidia-drivers</pkg>.</flag>
```
Comment (juippis) [inline app-laptop/thinkfan/metadata.xml]: These descriptions don't usually have dots in the end.

File: app-laptop/thinkfan/metadata.xml [hunk 3]
```diff
@@ -1,11 +1,27 @@
...
+		<flag name="yaml">Support the more flexible YAML config file format.</flag>
```
Comment (juippis) [inline app-laptop/thinkfan/metadata.xml]: These descriptions don't usually have dots in the end.

File: app-laptop/thinkfan/thinkfan-1.0.1.ebuild [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+LICENSE="GPL-3+"
+SLOT="0"
+IUSE="atasmart nvml +yaml"
+
+DEPEND="
+	atasmart? ( dev-libs/libatasmart:= )
+	yaml? ( dev-cpp/yaml-cpp:= )"
+
+RDEPEND="
+	${RDEPEND}
```
Comment (juippis) [inline app-laptop/thinkfan/thinkfan-1.0.1.ebuild]: Typo :)

File: app-laptop/thinkfan/thinkfan-1.0.1.ebuild [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+	nvml? ( x11-drivers/nvidia-drivers )
+"
+
+src_prepare() {
+	cmake-utils_src_prepare
+
+	# Don't install license file
+	# Don't use supplied init scripts
+	# Fix QA warning for doc dir
+	local mysedargs=(
```
Comment (juippis) [inline app-laptop/thinkfan/thinkfan-1.0.1.ebuild]: What's the point of this? You can just use sed here and shorten the ebuild for 3 lines. In my opinion, this makes it less obvious what's happening in here.

File: app-laptop/thinkfan/metadata.xml (hunk 1 above)
Comment (ConiKost) [reply app-laptop/thinkfan/metadata.xml]: That's new for me. No one complained so far about this.

File: app-laptop/thinkfan/metadata.xml (hunk 2 above)
Comment (ConiKost) [reply app-laptop/thinkfan/metadata.xml]: That's new for me. No one complained so far about this. Anyway, changed.

File: app-laptop/thinkfan/metadata.xml (hunk 3 above)
Comment (ConiKost) [reply app-laptop/thinkfan/metadata.xml]: That's new for me. No one complained so far about this. Anyway, changed.

File: app-laptop/thinkfan/thinkfan-1.0.1.ebuild (hunk 4 above)
Comment (ConiKost) [reply app-laptop/thinkfan/thinkfan-1.0.1.ebuild]: Thanks, fixed.

File: app-laptop/thinkfan/thinkfan-1.0.1.ebuild (hunk 5 above)
Comment (ConiKost) [reply app-laptop/thinkfan/thinkfan-1.0.1.ebuild]: I think, this gives a better readability, instead of having a long single line.

Comment (juippis) [reply app-laptop/thinkfan/thinkfan-1.0.1.ebuild]: I mean, 
```
sed -i -e 's/COPYING//' \
	-e 's/OPENRC_FOUND/DISABLE/g' \
	-e 's/SYSTEMD_FOUND/DISABLE/g' \
	-e "s/thinkfan)/${PF})/" \
	-CMakeLists.txt || die
```

is 4 lines shorter (3 if you drop first -e to its own line), doesnt add unnecessary complexity, and is more self-explanatory than your magic in here ;)

also looks like Zlogene already took this package, remember to do the `[please reassign]` trickstery to correctly assign it.

**Comment by Whissi** [comment]
Oh I missed that one. I bumped & fixed package on my own via 9001858ce0bae3908a65f4090c909fb99be6c5f7.

**Pull request closed without merging**